### PR TITLE
reflow comments in dof_tools_constraints.cc

### PR DESCRIPTION
**Pull request by davydden**

### CODE REVIEW

File: source/dofs/dof_tools_constraints.cc [hunk 1]
```diff
@@ -144,33 +142,33 @@ namespace DoFTools
...
+        // least as far as they have already been collected)
         return true;
       }
 
 
 
       /**
-       * When restricting, on a face, the degrees of freedom of fe1 to the
-       * space described by fe2 (for example for the complex case described
-       * in the @ref hp_paper "hp paper"), we have to select
```
Comment (davydden) [inline source/dofs/dof_tools_constraints.cc]: FYI `@ref`/`@note` prevents clang from reflowing.

**Comment by drwells** [comment]
Lets not merge these until #6822 is decided.

**Comment by davydden** [comment]
@drwells

> Lets not merge these until #6822 is decided.

I don't see how this is related to that PR neither https://github.com/dealii/dealii/issues/6814 . Reflow here is for incode comments that do not have any code-blocks inside them.

**Comment by drwells** [comment]
You're right, sorry. My tired eyes just picked everything that had 'reflow' in it.

**Pull request merged**